internal/broker/bind_create_test.go

```diff
@@ -378,6 +392,44 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		assert.Equal(t, firstInstanceSecondBindingDB.Kubeconfig, binding.Credentials.(map[string]interface{})["kubeconfig"])
 		assertClusterAccess(t, response, "secret-to-check-first", binding)
 	})
+
+	t.Run("should return error when maximum number of bindings is reached", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("should return error when maximum number of bindings is reached before adding a new one", func(t *testing.T) {
```

Reply:
`should return an error when attempting to add a new bind when the maximum number of bindingshas already been reached` Maybe this is better?